### DISCUSSION TITLE
SV06 plus?

### DISCUSSION THREAD

**Discussion by hannpet** [bassamanator/Sovol-SV06-firmware#20] [Q&A] [answered]
Can this also be used for the SV06 Plus model?
Which settings have to be adapted to make it work?
Thanks!

**Comment by bassamanator** [comment, marked as answer]
@hannpet I have a `sv06-plus` branch ready to go, however, it's mostly untested. This is not an issue, because the motherboard on the SV06 and the SV06-Plus are identical, as is the hotend PCB. The only thing that I would like to refine a bit, hopefully with your help, is the bed mesh size.

So to answer your question, the `sv06-plus` branch is ready to go, and in it's current state, no changes are necessary. 

Please read the docs fully on the plus branch, and I'll help you if you get stuck somewhere.

**Comment by hannpet** [comment]
That's great news.
I'd love to help, sadly I am not (yet) an owner of a sv06-plus!
I was trying to gather info about what kind of situation I'll be in when I buy one.
Awesome work, thanks!
